src/modules/new-request-form/fields/attachments/FileListItem.tsx

```diff
@@ -21,9 +22,18 @@ export function FileListItem({
   onRemove,
 }: FileListItemProps): JSX.Element {
   const { t } = useTranslation();
+  const closeButtonRef = useRef<HTMLButtonElement>(null);
+  const deleteButtonRef = useRef<HTMLButtonElement>(null);
```

Comment:
Hey @victor-g-vargas, I'm not sure if these are still necessary here --especially since they're typed as `HTMLButtonElement`, instead of `HTMLDivElement`.